sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2857,7 +2857,7 @@ pi_result hip_piEnqueueKernelLaunch(
   // Preconditions
   assert(command_queue != nullptr);
```

Comment:
I'm not sure what is the exact rule for using the new macro, but if it's meant to validate the existence of the object/pointer, you'd probably want it here as well as for `global_work_offset`?

Reply:
Correct, I did miss this one. I was trying to wrap several of the nullptr checks and invalid cases to align with PI error-reporting code. But I agree to separate those changes into a separate PR.

Reply:
👍 thank you.